doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,317 @@
+# Environment Variables
+
+The environment variables that Julia uses generally start with `JULIA`. If
+[`Base.versioninfo`](@ref) is called with `verbose` equal to `true`, then the
+output will list all environment variables for which `JULIA` appears in the
+name.
+
+## File locations
+
+### `JULIA_HOME`
+
+The absolute path of the directory containing the Julia executable, which sets
+the global variable [`Base.JULIA_HOME`](@ref). If `$JULIA_HOME` is not set,
+then `Base.JULIA_HOME` defaults to its compile-time value.
```

Comment:
this isn't quite right, you can move around the compiled executable and `Base.JULIA_HOME` will reflect the location it gets run from